nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-client-service-api/src/main/java/org/apache/nifi/elasticsearch/AuthorizationScheme.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nifi.elasticsearch;
+
+public enum AuthorizationScheme {
+    BASIC("Basic"),
```

Comment:
Is `BASIC` the most appropriate name here do we think, considering there are lots of [Elasticsearch Authentication Realms](https://www.elastic.co/guide/en/elasticsearch/reference/current/realms.html) that use Username & Password?

It's probably descriptive enough, but is it worth linking to the Elasticsearch docs in the controller documentation? Might not be required, just expect nifi users to figure out for themselves when they need to use `BASIC` to set a Username & Password if required by their Elasticsearch setup

Reply:
Since the API uses the Authorization header from the HTTP protocol, I thought it would make sense to follow that naming convention. I have changed the property descriptions, though. Could you check whether it is good now?